packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -122,7 +124,8 @@ export default class Meetings extends WebexPlugin {
        * @private
        * @memberof Meetings
        */
-      this.meetingInfo = new MeetingInfo(this.webex);
+
+      this.meetingInfo = Config.meetings.experimental.enableUnifiedMeetings ? new MeetingInfoV2(this.webex) : new MeetingInfo(this.webex);
```

Comment:
`this.meetingInfo` will need to be updated when the config value is changed, so we should probably have some kind of updateConfig() method in the SDK so that the SDK knows when config is changed. Or maybe config should be always constant and not change in runtime and we should have something else, like "SDK options" or "toggles" or something?

Reply:
when will we have a conclusion on this? We need that API to be defined so that we can do our bit of calling it.